awx/main/expect/isolated_manager.py

```diff
@@ -1,5 +1,4 @@
 import base64
-import cStringIO
```

Comment:
That's super unfortunate given the performance characteristics of cStringIO relative to the pure python implementation.